### PR TITLE
create landing page

### PR DESCRIPTION
**Pull request by Theoneste1**
#### What does this PR do?
create landing page on a user page with feature "ft-landingpage12"
#### Description of Task to be completed
create folder structure
create index.html
create style.css
#### How should this be manually tested?
clone the repo: git clone https://github.com/Theoneste1/My-Brand-Theoneste.git
get inside the project folder: cd My-Brand-Theoneste
switch to this branch git checkout ft-landingpage12
open index.html in your favourite browser and you will find it

#### Any background context you want to provide?
basic information that I want the people to know about me.
#### What are the relevant stories?
https://trello.com/c/VMmPDwAZ

#### Screenshots (if appropriate)
my first landing page

### CODE REVIEW

File: UI/landing-page/style.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
+*{
+    margin: 0;
+    padding: 0;
+    font-family: Georgia, 'Times New Roman', Times, serif;
+}
+section{
+    background-color:#252934;
+    /* min-height: 100vh; */
```
Comment (T2Wil) [inline UI/landing-page/style.css]: remove this comment line if it has no use.

**Comment by T2Wil** [comment]
Nice work @Theoneste1 ,

some important notes: 

make it easy for reviewer to test your code by making `how should this be tested` clearer.
you might write:
- clone the repo : `git clone https://github.com/Theoneste1/My-Brand-Theoneste.git`
- get inside the project folder: `cd My-Brand-Theoneste `
- switch to this branch `git checkout ft-landingpage-12345`
- open index.html in your favorite browser and voila

File: UI/landing-page/style.css (hunk 1 above)
Comment (Theoneste1) [reply UI/landing-page/style.css]: Removed, thanks. But I want to ask if it is allowed to include the comments, thanks!

Comment (T2Wil) [reply UI/landing-page/style.css]: The comments are allowed only if it makes your code readable.

**Pull request merged**